packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -35,14 +39,18 @@ def __init__(self, container, network, host, vip=None):
                 vip=vip,
                 container_type=container,
                 healthcheck_protocol=marathon.Healthcheck.MESOS_HTTP)
-        elif network == marathon.Network.USER:
+        elif network in [marathon.Network.USER, Network.IPv6]:
```

Comment:
This is pretty strange - valid values for `network` in this context are the marathon.Network enum values (`[HOST,USER,BRIDGE]`). I'm guessing the old behavior for plain old `USER` is that the `self.app['ipAddress']['networkName']` gets defaulted to `dcos`. I think it would make more sense to modify `MarathonApp`'s constructor to take this signature instead: `(self, container, network_type, network_name=None, host, vip=None):...` and then get rid of the `Network` enum in this file. if network_name is none then we don't set it, but it could have `'dcos6'` passed in.

Reply:
@drewkerrigan Thanks for reviewing it. Could you please check the last 2 commits?